src/oc_erchef/apps/migrator/test/migrator_tests.erl

```diff
@@ -0,0 +1,29 @@
+-module(migrator_tests).
+
+-compile([export_all]).
```

Comment:
:+1: on this test - we'll also want something similar as a sanity check prior to accepting a command to cutover, I think.